karlogonzales/ministocks-390
# Add the new widget view "resizable" to the screen space

# Description
A new widget view has been added to the list of available views. The user is now able to add the widget view "resizable" to the screen space along side of the currently available ones. 

# Fixes the Issue #8 
This widget view can be found in the list of available widget views at the end of the list of the Ministocks widgets.

# Testing the Feature
- Build the application either on a physical or an emulated mobile android phone
- Hold on an empty space on your screen until a menu appears
- Click on the widget button in order to display the list of available widgets
- Scroll down the list of widgets and find the Ministocks widget
- From the list of Ministocks widget, find the "resizable" widget view
- Hold onto the icon of the "resizable" widget view and add to the desired spot on your screen space

The widget should display a red rectangle with the text "Resizable".
